components/Sidebar/SidebarEpisode/SidebarEpisode.styles.ts

```diff
@@ -92,7 +76,8 @@ export const sidebarEpisodeTheme = (theme: Theme) =>
       MuiCardContent: {
         root: {
           color: theme.palette.text.primary,
-          fontWeight: theme.typography.fontWeightBold
+          fontWeight: theme.typography.fontWeightBold,
+          paddingBottom: '8px'
```

Comment:
Use theme for styling spacing: `paddingBottom: theme.typography.pxToRem(theme.spacing(1))`. Use `rem` units for spacing properties.

Reply:
Fixed!